lib/srv/desktop/tdp/proto.go

```diff
@@ -47,6 +52,12 @@ const (
 	TypeClientUsername   = MessageType(7)
 	TypeMouseWheel       = MessageType(8)
 	TypeError            = MessageType(9)
+	TypeMFAJson          = MessageType(10)
```

Comment:
```suggestion
	TypeMFAJSON         = MessageType(10)
```

Convention in Go is for acronyms to by all caps.